uploadNeuroDB/NeuroDB/DBI.pm

```diff
@@ -53,18 +53,26 @@ sub connect_to_db
     return $dbh;
 }
 
-sub getConfigSetting
-{
+sub getConfigSetting {
     my ($dbh, $name) = @_;
-    my ($message,$query,$where) = '';
+    my ($message, $query, $where) = '';
     my $value = undef;
 
     $where = " WHERE c.ConfigID=(Select cs.ID from ConfigSettings cs where cs.Name=?)";
     $query = " SELECT c.Value FROM Config c";
     $query = $query . $where;
     my $sth = $$dbh->prepare($query);
     $sth->execute($name);
-    if ( $sth->rows > 0 ) {
+
+    if ($sth->rows > 1){
```

Comment:
a tiny space missing in `){` it should be `) {`

Reply:
There is indeed a security risk in the regex thing. All special characters should be escaped in the constructed regex. Imagine one of the elements in @exclude is the string `rm -rf /` (with the backtick characters): it will be interpolated in the shell command and be run as a sub-command before the grep is run. I bet your filesystem won't be happy afterwards...

Just replace 

`my $excluded_regex = join('|', @$exclude);`

with 

`my $excluded_regex = join('|', map { quotemeta($_) } @$exclude);`

Reply:
@MounaSafiHarab @nicolasbrossard both fixed. Thanks!!